nodes/usb_cam_node.cpp

```diff
@@ -141,7 +141,7 @@ class UsbCamNode
     node_.param("image_height", image_height_, 480);
     node_.param("framerate", framerate_, 30);
     // possible values: yuyv, uyvy, mjpeg, yuvmono10, rgb24
```

Comment:
I guess we could also remove `mjpeg` from this comment.

Reply:
```suggestion
    // possible values: yuyv, uyvy, yuvmono10, rgb24
```